src/api/java/openiss/ws/soap/service/OpenISSSOAPServiceImpl.java

```diff
@@ -37,7 +39,7 @@
             }
 
             // file system routine
-            if (ServicePublisher.USE_FILESYSTEM) {
+            if (OpenISSConfig.USE_FAKENECT) {
```

Comment:
This file needs to be refactored completely to use the driver. e.g. getFrame should call directly kinect.getVideoImage or kinect.getDepthImage. Most of the logic should be removed here because it should be able to use the Kinect Wrapper or the Driver. 

- Please let me know if you find refactoring this last file complicated. I can do the refactoring on this file since I have a good understanding of how it would get refactored